### PR TITLE
centered the get-certified section

### PR DESCRIPTION
**Pull request by AnirudhPanda**
### BEFORE:
b

### AFTER:
a

### CODE REVIEW

**Comment by AnirudhPanda** [comment]
> Thanks for your pull request. Before we can look at your pull request, you'll need to sign a Contributor License Agreement (CLA).
> 
> 📝 **Please follow instructions at https://git.k8s.io/community/CLA.md#the-contributor-license-agreement to sign the CLA.**
> 
> It may take a couple minutes for the CLA signature to be fully registered; after that, please reply here with a new comment and we'll verify. Thanks.
> 
> * If you've already signed a CLA, it's possible we don't have your GitHub username or you're using a different email address.  Check your existing CLA data and verify that your [email is set on your git commits](https://help.github.com/articles/setting-your-email-in-git/).
> * If you signed the CLA as a corporation, please sign in with your organization's credentials at https://identity.linuxfoundation.org/projects/cncf to be authorized.
> * If you have done the above and are still having issues with the CLA being reported as unsigned, please log a ticket with the Linux Foundation Helpdesk: https://support.linuxfoundation.org/
> * Should you encounter any issues with the Linux Foundation Helpdesk, send a message to the backup e-mail support address at: [login-issues@jira.linuxfoundation.org](mailto:login-issues@jira.linuxfoundation.org)

Signed my CLA

**Comment by sftim** [comment]
/lgtm cancel

I just noticed that we're changing the text after the heading. I think we should just center the heading.
@AnirudhPanda do you agree?

**Comment by AnirudhPanda** [comment]
> /lgtm cancel
> 
> I just noticed that we're changing the text after the heading. I think we should just center the heading.
> @AnirudhPanda do you agree?

yeah, I saw that but I followed the code pattern which was used in the above block!
Capture

Should I update so that it covers just the `h2` tag??

File: content/en/training/_index.html [hunk 1]
```diff
@@ -79,36 +79,48 @@ <h2>Learn with the Linux Foundation</h2>
 
 <section id="get-certified">
   <div class="main-section padded">
-    <h2>Get Kubernetes Certified</h2>
-    <div class="col-container">
-      <div class="col-nav">
+    <center>
+      <h2>Get Kubernetes Certified</h2>
+    </center>
```
Comment (sftim) [inline content/en/training/_index.html:84]: Try this:
```suggestion
    <h2><center>Get Kubernetes Certified</center></h2>
```

_For another PR:_
Really we should update this page to use more Markdown and less HTML, and certainly no `<center>` elements!

**Comment by jihoon-seo** [comment]
Before: https://kubernetes.io/training/
After: https://deploy-preview-29214--kubernetes-io-main-staging.netlify.app/training/

File: content/en/training/_index.html (hunk 1 above)
Comment (AnirudhPanda) [reply content/en/training/_index.html:84]: Yeah for this PR I will modify this to the above!
@sftim should I create another PR to update the other `center` tags from HTML to Markdown?

**Comment by AnirudhPanda** [comment]
Can you please review it @sftim 
Thanks

**Review by sftim — CHANGES_REQUESTED** [review]
This PR _doesn't_ centre the headings for the boxes in “Get Kubernetes Certified”.

The outcome I think we want is that in each box its header is centred but the remaining text is justified left.

File: content/en/training/_index.html (hunk 1 above)
Comment (sftim) [reply content/en/training/_index.html]: @AnirudhPanda would you be willing to open an issue about converting this page to Markdown (there might be one already, mind - if so, that's OK, we can use the existing issue).

**Comment by AnirudhPanda** [comment]
> This PR _doesn't_ centre the headings for the boxes in “Get Kubernetes Certified”.
> 
> The outcome I think we want is that in each box its header is centred but the remaining text is justified left.


@sftim Like this??
ref

File: content/en/training/_index.html [hunk 2]
```diff
@@ -32,81 +34,100 @@ <h2>Build your cloud native career</h2>
...
+        <center>
+          <h5>
+            <b>Introduction to Kubernetes <br> &nbsp;</b>
+          </h5>
+          <p>Want to learn Kubernetes? Get an in-depth primer on this powerful system for managing containerized
+            applications.</p>
+          <br>
+          <a href="https://www.edx.org/course/introduction-to-kubernetes" target="_blank" class="button">Go to
+            Course</a>
+        </center>
```
Comment (sftim) [inline content/en/training/_index.html:48]: Try this:
```suggestion
        <h5>
          <center><b>Introduction to Kubernetes</b></center> <br> &nbsp;
        </h5>
        <p>Want to learn Kubernetes? Get an in-depth primer on this powerful system for managing containerized
          applications.</p>
        <br>
        <center>
        <a href="https://www.edx.org/course/introduction-to-kubernetes" target="_blank" class="button">Go to
          Course</a>
        </center>
```

**Comment by sftim** [comment]
Hi @AnirudhPanda 

Did that last feedback make sense?

**Comment by AnirudhPanda** [comment]
> Hi @AnirudhPanda
> 
> Did that last feedback make sense?

Tried it but it changes the page in a weird way!

**Comment by AnirudhPanda** [comment]
To be honest, the page looks pretty neat when the headings are centered!
Have you thought if we can use `justify` instead of `center` on paragraphs

**Comment by sftim** [comment]
The _best_ option is to switch that page to use CSS for styling and HTML classes (or CSS selectors) to identify what to style how.

**Comment by AnirudhPanda** [comment]
> The _best_ option is to switch that page to use CSS for styling and HTML classes (or CSS selectors) to identify what to style how.

Yeah that sounds good

**Comment by natalisucks** [comment]
Hey there, @AnirudhPanda – just wanted to check how you're getting on with @sftim's suggestion of switching to CSS for styling/HTML classes. Thanks so much already for your contributions thus far!

**Comment by AnirudhPanda** [comment]
> Hey there, @AnirudhPanda – just wanted to check how you're getting on with @sftim's suggestion of switching to CSS for styling/HTML classes. Thanks so much already for your contributions thus far!

Should I change the CSS file instead of the markdown? @sftim

**Comment by jimangel** [comment]
Hey @AnirudhPanda thanks for the PR! Are you still interested in working on it? Given the age of this PR without any further action, I'm going to close it, but feel free to reopen if you start working again on it. 

> Should I change the CSS file instead of the markdown?

It is my understanding that this is what @sftim was requesting, if you need help on the "how" or "where" feel free to reach out on [slack](https://kubernetes.slack.com/messages/sig-docs)! Thanks

/close

**Pull request closed without merging**